app/presenters/image_upload_response_presenter.rb

```diff
@@ -91,7 +92,11 @@ def failed_fingerprints
   end
 
   def show_selfie_failures
-    @form_response.extra[:liveness_checking_required] == true
+    @form_response.extra[:liveness_enabled] == true
+  end
+
+  def selfie_status
```

Comment:
I'm not sure I understand why we would return `true` here if the response doesn't respond to `selfie_status`. Do you mind clarifying why that is?

Reply:
Fixed! This should return the fallback `:not_processed`